spyder/plugins/application/widgets/status.py

```diff
@@ -0,0 +1,100 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright © Spyder Project Contributors
+# Licensed under the terms of the MIT License
+# (see spyder/__init__.py for details)
+
+"""
+Status widget for Spyder updates.
+"""
+
+# Standard library imports
+import logging
+
+# Third party imports
+from qtpy.QtCore import Slot
```

Comment:
To enable showing a menu from the status bar and adding a new `Signal`:
```suggestion
from qtpy.QtCore import QPoint, Signal, Slot
from qtpy.QtWidgets import QMenu
```